source/2019-04-28-the-ember-times-issue-96.md

```diff
@@ -44,21 +46,29 @@ But there's a new hope, called [Milestones](https://twitter.com/__dfreeman/statu
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
+## [Edward Faulkner on Embroider, Ember CLI's Modern Build System 🎙️](https://embermap.com/podcast/edward-faulkner-on-embroider-ember-cli-s-modern-build-system)
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+At [EmberConf 2019](https://youtu.be/zYwdBcmz6VI?t=2855), we got a surprise announcement of [Embroider](https://github.com/embroider-build/embroider), the **next-gen build pipeline for Ember applications**. 🧡
+
+[EmberMap sat down with Edward Faulkner](https://embermap.com/podcast/edward-faulkner-on-embroider-ember-cli-s-modern-build-system), the creator of Embroider, to provide us more information.
+
+Some key messages are:
+
+- Embroider embraces ES modules and pull-based builds. It includes **only the code that your app needs.** 💯
```

Comment:
There is some really weird stuff going on with bulleted text. Markdown for bold does not show up in bulleted text. 

![image](https://user-images.githubusercontent.com/1372946/57167128-79b42980-6db1-11e9-80f6-385c4fae8a00.png)

Since the blog backend is being rewritten slowly but surely, I'm not too worried about it, just strange.

Reply:
https://github.com/ember-learn/ember-blog/issues/101 to investigate later